### PR TITLE
Configure repository arguments

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Component/Repository/ArgumentParser.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+namespace Sylius\Component\Resource\Repository;
+
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\Security\Core\Authentication\Token\NullToken;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+
+final class ArgumentParser
```
Comment (lchrusciel) [inline src/Component/Repository/ArgumentParser.php]: Let's add an interface here. Also, Do we need different parsers with - https://github.com/Sylius/SyliusResourceBundle/pull/645#discussion_r1154786501?

Comment (loic425) [reply src/Component/Repository/ArgumentParser.php]: done

**Review by lchrusciel — COMMENTED** [review]
requires rebasing

**Comment by loic425** [comment]
> requires rebasing

done

**Comment by loic425** [comment]
Replaced by #681

**Pull request closed without merging**